container/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:21.7.2-bookworm-slim as frontend-builder
+
+WORKDIR /app
+
+COPY ./frontend/package.json frontend/package-lock.json ./
+RUN npm install
+
+COPY ./frontend ./
+RUN npm run build
+
+
+FROM python:3.12-slim as runtime
+
+WORKDIR /app
+ENV PYTHONPATH '/app'
+ENV RUN_AS_DEVIN=false
+ENV USE_HOST_NETWORK=false
+ENV SSH_HOSTNAME=host.docker.internal
+ENV WORKSPACE_BASE=/opt/workspace_base
+RUN mkdir -p $WORKSPACE_BASE
+
+RUN apt-get update -y
+RUN apt-get install -y curl make git build-essential
```

Comment:
apt update and install should go in the same RUN statement. 
https://docs.docker.com/develop/develop-images/instructions/#apt-get

Reply:
Good call!